timer.js "time count" unit test may be too tight?

Running `npm test` several times, I saw this failure a few times:
`Uncaught AssertionError: expected 0.009798363 to be within 0.01..0.015`

MacBook Air i7, uptime reports `load averages: 1.94 2.50 2.82` so maybe my system is just bogged down.

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "http-test-servers": "0.0.12",
+    "injectr": "^0.5.1",
     "jshint": "2.5.10",
     "mocha": "",
     "should": "3.3.2"

--- a/test/index.js
+++ b/test/index.js
@@ -5,7 +5,7 @@ var Mocha = require('mocha'),
 
 var runAcceptanceTests = function(callback){
 	console.log('Running acceptance tests...');
-	var acceptance = new Mocha({timeout: 60000, reporter: 'list' });
+	var acceptance = new Mocha({timeout: 60000, reporter: 'spec' });
 
 	fs.readdirSync('./test/acceptance/')
 		.filter(function(file){ return file.substr(-3) === '.js';})
@@ -16,7 +16,7 @@ var runAcceptanceTests = function(callback){
 
 var runUnitTests = function(callback){
 	console.log('Running unit tests...');
-	var unit = new Mocha({ reporter: 'list'});
+	var unit = new Mocha({ reporter: 'spec'});
 
 	fs.readdirSync('./test/unit/')
 		.filter(function(file){ return file.substr(-3) === '.js';})

--- a/test/unit/timer.js
+++ b/test/unit/timer.js
@@ -1,35 +1,39 @@
 'use strict';
 
+var injectr = require('injectr');
 var should = require('should');
-var Timer = require('./../../lib/timer');
 
 describe('Timer class', function(){
 
-  it('should correctly perform a time count', function(done){
-    var timer = new Timer();
-
-    timer.start();
-    setTimeout(function() {
-      timer.stop();
-      timer.time.should.be.within(0.09, 0.12);
-      done();
-    }, 100);
+  var Timer = injectr('./../../lib/timer.js', {}, {
+    process: {
+      hrtime: function(start){
+        return start ? [10, 0] : [235389, 949793216];
+      }
+    }
   });
 
-  it('should correctly perform a stop + restart action', function(done){
+  it('should correctly perform a time count', function(){
     var timer = new Timer();
 
+    // Process.hrtime is mocked to return a 10 seconds timer
     timer.start();
-    setTimeout(function() {
-      timer.stop();
-      setTimeout(function() {
-        timer.start();
-        setTimeout(function() {
-          timer.stop();
-          timer.time.should.be.within(0.19, 0.22);
-          done();
-        }, 100);
-      }, 100);
-    }, 100);
+    timer.stop();
+
+    timer.time.should.be.eql(10);
+  });
+
+  it('should correctly perform a stop + restart action', function(){
+    var timer = new Timer();
+
+    // Process.hrtime is mocked to return a 10 seconds timer
+    timer.start();
+    timer.stop();
+
+    // Process.hrtime is mocked to return a 10 seconds timer
+    timer.start();
+    timer.stop();
+
+    timer.time.should.be.eql(20);
   });
 });

